src/client/App.jsx

```diff
@@ -11,80 +11,83 @@ function App() {
   useEffect(() => {
     fetch(`${apiUrl}/movie`)
       .then(res => res.json())
-      .then(res => setMovies(res.data));
+      .then(res => {
+        console.log(res.data)
+        setMovies(res.data)
+      });
   }, []);
 
   const handleRegister = async ({ username, password }) => {
 
     const options = {
       method: 'POST',
       headers: {
-          'content-type': 'application/json',
+        'content-type': 'application/json',
       },
       body: JSON.stringify({
-          username: username,
-          password: password
+        username: username,
+        password: password
       }),
-  }
+    }
 
-  fetch('http://localhost:4000/user/register', options)
+    fetch(`${apiUrl}/user/register`, options)
       .then(res => res.json())
       .then(json => console.log('Registered User:' + json.data.username))
       .catch(() => {
-          console.log('server error')
+        console.log('server error')
       })
-  };
+  }
 
   const handleLogin = async ({ username, password }) => {
-    
+
     const options = {
       method: 'POST',
       headers: {
-          'content-type': 'application/json',
+        'content-type': 'application/json',
       },
       body: JSON.stringify({
-          username: username,
-          password: password
+        username: username,
+        password: password
       }),
-  }
+    }
 
-  fetch('http://localhost:4000/user/login', options)
+    fetch(`${apiUrl}/user/login`, options)
       .then(res => res.json())
       .then(json => {
         console.log('logged in:' + json.data)
         localStorage.setItem('jwt', json.data)
       })
   };
-  
+
   const handleCreateMovie = async ({ title, description, runtimeMins }) => {
 
     const options = {
       method: 'POST',
       headers: {
-          'content-type': 'application/json',
-          'authorization': 'Bearer ' + localStorage.getItem('jwt')
+        'content-type': 'application/json',
+        'authorization': 'Bearer ' + localStorage.getItem('jwt')
```

Comment:
noticed this the other day - what does 'Bearer' achieve in this string? Mine seemed to work with just the token.

Reply:
After a quick read online, it seems there are different types of authorisation. "Many web servers support multiple methods of authorization. In those cases sending just the token isn't sufficient." - taken from stackexchange.